tests/screenshot/apps/components/ContextualPoppDecorator.js

```diff
@@ -0,0 +1,10 @@
+import ContextualPopupDecorator from '../../../../ContextualPopupDecorator';
+import React from 'react';
+
+const ContextualPopupDecoratorTests = [
+	<ContextualPopupDecorator />,
+	<ContextualPopupDecorator direction="above" />,
+	<ContextualPopupDecorator noArrow />
```

Comment:
Decorator should be used as a tag. It is not a component but a HoC. Please check usage from https://github.com/enactjs/agate/blob/develop/samples/qa-a11y/src/views/ContextualPopupDecorator.js#L12 and screenshot result.